R/wrappers.R

```diff
@@ -0,0 +1,268 @@
+# Dash 1.0 Layout Wrapper Functions (adapted from https://github.com/daattali/dash2)
+
+#' Create a Dash application
+#'
+#' This is a convenience function that returns a [`dash::Dash`] R6 object.
+#' For advanced usage, you can use the object as an R6 object directly instead
+#' of the functions provided by the `{dash2}` package.
+#'
+#' @param title _(character)_ The browser window title.
+#' @param update_title _(character)_ The browser window title while a callback
+#' is being processed. Set to `NULL` or `”"` if you don't want Dash to
```

Comment:
```suggestion
#' is being processed. Set to `NULL` or `""` if you don't want Dash to
```
Again in the copy that stayed 😉

Reply:
Thanks :hocho:  in ebdbe37